test/local/main.tf

```diff
@@ -9,7 +9,7 @@ terraform {
 provider "bindplane" {
   remote_url = "http://localhost:3001"
   username = "admin"
-  password = "admin"
+  password = "password"
```

Comment:
Init no longer allows you to use "admin" as a password, so I use "password" for local testing in short lived dev environments.